NConfiguration.Tests/NConfiguration.Tests.csproj

```diff
@@ -264,7 +265,7 @@
         </Reference>
       </ItemGroup>
     </When>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
+	<When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">

```

Comment:
This file is indented by whitespaces.
Replace indenting `\t` by whitespaces.

Reply:
Fixed, thank you